govcd/org_test.go

```diff
@@ -191,9 +191,18 @@ func (vcd *TestVCD) Test_Admin_GetVdcByName(check *C) {
 // if the error is not nil.
 func (vcd *TestVCD) Test_CreateVdc(check *C) {
 	if vcd.skipAdminTests {
-		check.Skip("Configuration org != 'Sysyem'")
+		check.Skip("Configuration org != 'System'")
 	}
 
+	if vcd.config.VCD.ProviderVdc.Name == "" {
+		check.Skip("No Provider VDC name given for VDC tests")
+	}
+	if vcd.config.VCD.ProviderVdc.StorageProfile == "" {
+		check.Skip("No Storage Profile given for VDC tests")
+	}
+	if vcd.config.VCD.ProviderVdc.NetworkPool == "" {
+		check.Skip("No Network Pool given for VDC tests")
```

Comment:
Shouldn't all three messages above say "for Org tests" instead of "for VDC tests"?

Reply:
No. It's the VDC creation test that gets skipped. This test is in `org_test.go` because we have the creation cases in the parent's entity test file. For example, the Org creation test is in `system_test.go`.